### DISCUSSION TITLE
Запрет на вход игрока

### DISCUSSION THREAD

**Discussion by SobakaBarabaka74** [Sokol-Games-GmbH/PixelCombats.PublicApi#239] [Ideas]
**Сделать функционал который будет запрещать вход игрокам на сервер по указанному контексту.**

***Отключение всем входа в комнату кроме 1 игрока через интерфейс.***
```js
ConnectedHandler.GetContext().ConnectedEnable.Value = false;

// или
  
Teams.Get("Red").ConnectionHandler.ConnectionEnable.Value = false;  // как пример отключить вход на сервер тем игрокам, кто побывал в команде Red
```

***Отключение входа по свойству***
```js
try {
   
    Players.Get("id").ConnectionHandler.ConnectionEnable.Value = false;

    // или
    Players.Get("id").ConnectionEnable.Value = false;

} catch(err) {
  // null
}
```

Всё это дело конечно будет сохранятся.

**Comment by kkohno** [comment]
мысль интересная, а что насчет гарантий безопасности?

**Reply by SobakaBarabaka74** [reply]
Каких гарантий? Создатель режима пускай сам решает это. Кому не понравится - те не будут играть на том режиме
